docs/naps/index.md

```diff
@@ -0,0 +1,20 @@
+(nap_list)=
+# napari Advancement Proposals (NAPs)
+
+NAPs document any major changes or proposals to the napari project.
+
+## Sections
+
+```{toctree}
+---
+maxdepth: 1
+---
+0-nap-process
+```
+
+```{toctree}
+---
+hidden:
+---
+template
+```
```

Comment:
This can be decided at a later date. I think for a start they could just be in order of acceptance and just contain number and title, and we can elaborate as we go.